modules/plugins/languages/markdown.nix

```diff
@@ -133,6 +133,18 @@ in {
           };
         };
       };
+      markview-nvim = {
+        enable =
+          mkEnableOption ""
+          // {
+            description = ''
+              [markview.nvim]: https://github.com/OXY2DEV/markview.nvim
+
+              A hackable markdown, Typst, latex, html(inline) & YAML previewer
```

Comment:
the `[markview.nvim]: https://github.com/OXY2DEV/markview.nvim` part defines a reference. It won't be *actually* rendered unless you refer to it. Either use it, or remove it. Refer to render-markdown-nvim plugin toggle on how to use it.